### PR TITLE
APIGOV-21813 Create February Release Notes

### PR DESCRIPTION
**Pull request by lbadenhop**
Thank you for your contribution to the Amplify-Central repo.

## Describe the changes

Created Release Notes for all components outlined in Jira

## Checklist for contributors

Before submitting this PR, please make sure:

* [x] You have read the [contribution guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/)
* [x] You have signed the [Axway CLA](https://cla.axway.com/)
* [x] You have verified the technical accuracy of your change
* [x] You have verified that your change does not expose any sensitive information (passwords, keys, etc.)
* [x] You have followed the [Markdown guidelines](https://axway-open-docs.netlify.com/docs/contribution_guidelines/writing_markdown/)  (unless this is is a Netlify CMS contribution)

_Put an x in the boxes that apply. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your change._

### CODE REVIEW

**Comment by lbadenhop** [comment]
@cbordier Ready for you to add content

File: content/en/docs/amplify_relnotes/20220228_analytics_relnotes.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+* Comparing insights: each screen has a compare capability to visualize the trend with the previous period.
+* Leaderboard view: help to verify your API performance: the most/least used, the most/least performant.
+
+More information can be found [here](/docs/get_actionable_insights)
+
+## Fixed issues
+
+The following issues have been fixed in this release:
+
+* **Application usage no more visible** Previously, when the report metrics feature has been activated, the App usage usage cannot be computed correctly. Now, the app usage is correctly computed based on the transactions sent by the Traceability agent. Be aware that TA can sample
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20220228_analytics_relnotes.md:22]: I would rephrase this to: Before this fix, when the report metrics feature was turned on, the App usage was not reported correctly. After the fix, the  App usage is correctly computed based on the transactions reported by the Traceability agent.  
**Be aware of the TA can sample** seems to be an incomplete phrase.

File: content/en/docs/amplify_relnotes/20220228_catalog_relnotes.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+linkTitle: Asset Catalog February 2022
+weight: 90
+date: 2022-02-25
+description: Add description
+---
+## New features and enhancements
+
+The following new features and enhancements are available in this update:
+
+* User can view, create, edit, release, deprecate, archive and delete an asset.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20220228_catalog_relnotes.md:12]: - Developers can create, view, edit and delete an asset, 

- Lifecycle management that covers the following stages: 
  - release
  - deprecate
   - archive 
- The asset logical name can be computed from the asset title or auto-generated when the asset title is missing. 
- When an asset version is released, a Major, Minor or Patch release type can be selected. 
- Make changes to an asset after has been released by using the 'Create new version' button. This will allow users to make changes to the asset in DRAFT mode, without affecting previous assets versions. 
- An asset can have multiple active versions and only one DRAFT.
- Assets that are packaged with a product can not be archived. When users attempt to remove an asset that is used in a product, the WebUI will show an error message informing that the asset is used in a product and the delete is not permitted.

File: content/en/docs/amplify_relnotes/20220228_foundry_relnotes.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+description: Add description
+---
+## New features and enhancements
+
+The following new features and enhancements are available in this update:
+
+* User can view, create, edit, release, deprecate, archive and delete a product
+* A product is automatically created with a free plan
+* The product logical name is derived from the product title.
+* A product can be associated with multiple assets
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20220228_foundry_relnotes.md:15]: - Multiple assets can be included in a product.

File: content/en/docs/amplify_relnotes/20220228_foundry_relnotes.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+* Product documentation is using Markdown format. Multiple articles can be added to a product
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20220228_foundry_relnotes.md:16]: - Product can be documented using markdown content. Multiple markdown articles can be added to a product.

File: content/en/docs/amplify_relnotes/20220228_foundry_relnotes.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+* Product can be released (Major / Minor / Patch) and published to the Marketplace
+* A new version of a released product can be created using the `Create Draft` button
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20220228_foundry_relnotes.md:18]: Changes can be made to a product by using the 'Create Draft' button. Users can continue to make changes to the product as long as the product is in DRAFT. To make the changes available in the Marketplace, a new version has to be released.

File: content/en/docs/amplify_relnotes/20220228_foundry_relnotes.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+* A product support multiple active versions and only one draft (the potential next release)
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20220228_foundry_relnotes.md:19]: -  A product support multiple active versions and only one draft.

File: content/en/docs/amplify_relnotes/20220228_foundry_relnotes.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+The following issues have been fixed in this release:
+
+* None
+
+## Known limitations
+
+The following limitations exist in this update:
+
+* Documentation update is not yet available.
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20220228_foundry_relnotes.md:31]: - Adding a new document to a product from the WebUI  is not yet available. 
- Edit a plan is not yet available.

File: content/en/docs/amplify_relnotes/20220228_foundry_relnotes.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+## New features and enhancements
+
+The following new features and enhancements are available in this update:
+
+* User can view, create, edit, release, deprecate, archive and delete a product
+* A product is automatically created with a free plan
+* The product logical name is derived from the product title.
+* A product can be associated with multiple assets
+* Product documentation is using Markdown format. Multiple articles can be added to a product
+* Product can be released (Major / Minor / Patch) and published to the Marketplace
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20220228_foundry_relnotes.md:17]: - A product can be released with a Major, Minor or Patch version. 
- A product can be published to the Marketplace after a version has been released. All subsequent product versions will be automatically exposed in the Marketplace for a published product.

File: content/en/docs/amplify_relnotes/20220228_marketplace_relnotes.md [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+linkTitle: Marketplace February 2022
+weight: 90
+date: 2022-02-25
+description: Add description
+---
+## New features and enhancements
+
+The following new features and enhancements are available in this update:
+
+* Marketplace settings:
```
Comment (georgiana-caras) [inline content/en/docs/amplify_relnotes/20220228_marketplace_relnotes.md:12]: - Platform administrators can configure the following Marketplace properties, from the Marketplace settings page found under their Organization profile menu. 

  - Define the marketplace sub-domain. This is required for the Marketplace to become enabled. 

- Platform administrators can change the look and feel of their Marketplace from the Marketplace Appearance page. The following branding options are available: 
  - branding of the logo and favicon
  - changing the color theme (Default colors, card colors and semantic colors)
  - changing the button appearance
  - changing the text font family and kerning
Refer to <https://platform.axway.com/org/marketplace/settings>

- Marketplace is available in public mode.

**Review by georgiana-caras — CHANGES_REQUESTED** [review]

**Comment by lbadenhop** [comment]
@cbordier Please do a final review before I merge. Thanks.

**Review by cbordier — APPROVED** [review]
I approve the changes.
@lbadenhop
